packages/kbn-scout/src/playwright/page_objects/maps_page.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+export class MapsPage {
+  constructor(private readonly page: ScoutPage) {}
+
+  /**
+   * @deprecated see: https://github.com/elastic/kibana/pull/204607#discussion_r1930723011
+   */
+  async goto() {
+    await this.page.gotoApp('maps');
+  }
```

Comment:
Let's design Page Object as Nathan suggested, we might not come back to it ourself and forget it.

Reply:
first attempt: https://github.com/elastic/kibana/pull/204607/commits/3e1826699e827b415b67e6ee21f6ed146be9a548